src/fragments/lib/auth/flutter/delete_user/10_delete_user.mdx

```diff
@@ -0,0 +1,14 @@
+<Callout>
+  Amplify-flutter currently offers this API only on the iOS platform, in
+  accordance with iOS App Store policy changes. It is recommended that you
+  display widgets to call this API only on the the iOS platform.
+</Callout>
```

Comment:
*nit:* Should we link to a feature request for adding Android support? I could imagine this message being frustrating for a customer.